src/seismometer/controls/explore.py

```diff
@@ -526,7 +526,7 @@ def __init__(
         all_metrics = ["pprev", "tpr", "fpr", "fnr", "ppr", "precision"]
```

Comment:
Line 517: thresholds is in the docstring but score_threshold is listed as the parameter

Reply:
They also have different types so I'm not sure if they were meant to be the same thing

Reply:
fixed, it should be score_threshold, and the typing was updated.